src/test/java/edu/oakland/test/database/DatabaseCommManagerTest.java

```diff
@@ -0,0 +1,94 @@
+package edu.oakland.test.database;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.database.DatabaseCommManagerImplementation;
+import java.lang.IllegalArgumentException;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("DatabaseCommManagerImplementation Unit Tests")
+public class DatabaseCommManagerImplementationTest {
```

Comment:
One more note I missed earlier:
```suggestion
public class DatabaseCommManagerImplementationTests {
```